rootston/output.c

```diff
@@ -17,13 +17,40 @@ static inline int64_t timespec_to_msec(const struct timespec *a) {
 	return (int64_t)a->tv_sec * 1000 + a->tv_nsec / 1000000;
 }
 
+/**
+ * Computes the surface size homogeneous to the global compositor coordinates.
+ */
+static void get_surface_size(struct wlr_surface *surface, double *width,
+		double *height) {
+	double surface_scale = surface->current->scale;
+	*width = (double)surface->current->buffer_width / surface_scale;
+	*height = (double)surface->current->buffer_height / surface_scale;
```

Comment:
Isn't this already present in `buffer->current->width|height`?

Reply:
You're right, fixed.